css/style.css

```diff
@@ -1,11 +1,174 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+ 
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    position: relative;
+}
+ /* Header  */
+ header {
+    font-size: 24px;
+    width: 100%;
+    height: 20px;
+    padding: 40px;
+    margin: 20px;   
```

Comment:
This spacing is being counted on top of the `width: 100%` which is why if you scroll right, there is additional space on the side of your page. 
I would remove the margin as you shouldn't need it. 
For the padding, you can add the rule `box-sizing: border-box` to your header which will force the padding to be _included_ in the `width: 100%`. It's quite common to add this rule to every element, by using the star selector:
```
* {
  box-sizing: border-box;
}
```

Reply:
I changed it, thank you.